torchcontrib/optim/__init__.py

```diff
@@ -0,0 +1 @@
+from .swa import SWA, bn_update
```

Comment:
Please make `bn_update` a static member of `SWA` (so someone can do `SWA.bn_update(...)`). Otherwise it pollutes the optim namespace, and its name doesn't indicate that it's really specific to this optimizer

Reply:
done